docs/2.build/1.chain-abstraction/multichain-gas-relayer/relayer-gas-example.md

```diff
@@ -11,24 +11,34 @@ In this article you'll learn how to run end-to-end tests on the entire Multichai
 
 :::info Required tools
 
-For this tutorial, you'll need to have installed:
+For this tutorial, you need to have installed:
 
+ - [Python](https://python.org) >=3.10
+ - [Rust](https://rust-lang.org)
  - [Multichain Relayer Server](https://github.com/near/multichain-relayer-server)
  - [Gas Station Event indexer](https://github.com/near/gas-station-event-indexer)
- - [NEAR CLI RS](https://github.com/near/near-cli-rs)
+ - [NEAR CLI RS](https://github.com/near/near-cli-rs): Make sure to configure it with the correct network and account.
 
 :::
 
 ## Setup
 
-Before you start testing, set up your local environment and install the Relayer server, the Event indexer and NEAR CLI.
+Before you start testing, set up your local environment and install the [Relayer server](multichain-server.md), the [Event indexer](https://github.com/near/gas-station-event-indexer) and [NEAR CLI](https://github.com/near/near-cli-rs).
 
 :::info Alpha stage
```

Comment:
Code audit it complete. Think we can remove this section

Reply:
Removed ✅